src/components/video.js

```diff
@@ -118,37 +149,34 @@ export default function Video(props) {
               })}
             </div>
           </div>
-          <div
-            className='next_song control_button'
-            onClick={() => {
-              changeQueue(queue.slice(1));
-            }}
-          >
+          <div className='next_song control_button' onClick={playNextVideo}>
             <Icon.ChevronRight />
           </div>
         </div>
       </div>
 
       <div className='queue'>
-        <h1 className='title'>Up next</h1>
-        {queue.slice(1).map((id) => (
-          <Card
-            id={id}
-            key={id + sampleData.yid}
-            className='queueCard'
-            redirect={false}
-            onClick={() => {
-              changeQueue(
-                queue.slice(
-                  queue.findIndex((item) => {
-                    return id === item;
-                  })
-                )
-              );
-              props.history.push('video?=' + id);
-            }}
-          />
-        ))}
+        <h1 className='title'>Queue</h1>
+        {queue.map((id, index) => {
+          let selectClass = '';
+          if (index === currentIndex - 1) {
+            selectClass = ' previous';
```

Comment:
Is the `previous` classed used anymore? Do leave it here if you feel it could be useful later though.

Reply:
I have removed the previous class, the new scroll into view doesn't use it and we can insert it if the need arises later.